impress-org/givewp
# Fix: Update styles for archive campaign modal UI

## Description

This PR updates the archive campaign modal UI to match the designs in Zeplin. I also added hover states to both the cancel and archive campaign button. Updated our design system to make good use of the neutral colors.

## Affects

Affects the archive campaign modal in campaigns.

## Visuals

Before
https://www.loom.com/share/53b768c90434437d98e051f201e0043d?sid=33dc9237-5ef4-43e4-9339-fad58f813c99

After
https://www.loom.com/share/293a55f6123b42fdb9a5eb53b1496166?sid=c84b8d6c-94f0-4519-803d-debf2e7bd8ed

## Testing Instructions

1. Open an active campaign
2. Click on the more button and select archive campaign
3. Check the updated modal UI

## Pre-review Checklist
